taskfiles/lint.yaml

```
@@ -135,6 +135,9 @@
     cmds:
       - task: ":utils:cpp-lint:clang-format"
         vars:
+          EXCLUDE_PATTERNS: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)